src/zcash/ZcashEngine.ts

```diff
@@ -42,13 +42,16 @@ export class ZcashEngine extends CurrencyEngine<
   availableZatoshi!: string
   initialNumBlocksToDownload!: number
   initializer!: ZcashInitializerConfig
-  alias!: string
   progressRatio!: number
   queryMutex: boolean
   makeSynchronizer: (
     config: ZcashInitializerConfig
   ) => Promise<ZcashSynchronizer>
 
+  // Synchronizer management
```

Comment:
Please move `synchronizer` to this section, and make it `?:` instead of `!:`. This will force you to check for `null` before accessing the synchronizer, since we have even less guarantee than before that the synchronizer will exist.